### DISCUSSION TITLE
Haptic feedback or UI feedback on tapping link

### DISCUSSION THREAD

**Discussion by Tarang** [gonzalezreal/swift-markdown-ui#272] [Q&A]
Hi,

Is there a way to get UI feedback when tapping a link? If it changes colour or has haptic feedback it would help the feeling of it being static

**Comment by ryanbooker** [comment]
I would love this too. A "tapped" or "active" state for the link and the ability to set some haptic feedback.

**Reply by jonmindtrip** [reply]
Sign me up for this also - I'm not sure how this would be possible though - anyone have any ideas?
